Scripts/script-FilterByWhitelist.yml

```diff
@@ -0,0 +1,57 @@
+commonfields:
+  id: FilterByWhitelist
+  version: -1
+name: FilterByWhitelist
+script: |
+  lst = demisto.executeCommand("getList", {"listName": demisto.args()["whitelistname"]})
+
+  if isError(lst[0]):
+      demisto.results(lst)
+      sys.exit(0)
+
+  whitelist = lst[0]['Contents']
+  item = demisto.args().get('value', '')
+
+  if item == '':
+      demisto.results("No item was given")
+      sys.exit(0)
+
+  ec = {}
+  contents = {}
+  hr = item + " is not in the whitelist"
+
+  if item in whitelist:
+      hr = item + " is in whitelist"
+      ec["Whitelist.In"] = item
+      contents["inWhitelist"] = item
+  else:
+      ec["Whitelist.NotIn"] = item
+      contents["notInWhitelist"] = item
+
+  demisto.results({'ContentsFormat': formats['json'],
+                       'Type': entryTypes['note'],
+                       'Contents': contents,
+                       'HumanReadable' : hr,
+                       'HomanReadableFormat' : formats['markdown'],
+                       'EntryContext': ec})
+
+  sys.exit(0)
+type: python
+tags:
+- whitelist
+comment: Check whether the given item is whitelisted. Default list is the Demisto Indicators Whitelist
+enabled: true
+args:
+- name: value
+  default: true
+  description: The value to look for on the whitelist
+- name: whitelistname
+  description: Name of whitelist to check
+  defaultValue: Indicators Whitelist
+outputs:
+- contextPath: Whitelist.In
+  description: List of items in the whitelist
+- contextPath: Whitelist.NotIn
+  description: List of items not in the whitelist
```

Comment:
I would check the wording with @kirbles19  - for all descriptions

Reply:
- Line 35 should be `HumanReadableFormat` not `HomanReadableFormat`.
- Line 42: Check whether the specified item is whitelisted. Default list is the Demisto Indicators Whitelist.
- Line 47: Value might be the argument name, but what is it a value of? an IP? or are there multiple valid values?
- Line 53: List of items on the whitelist (Try to stay consistent with in/on. We used on in Line 47, so better to keep it the same throughout)
Line 55: List of items not on the whitelist

Reply:
@kirbles19 regarding line 47 - it could be anything.
The main usage of this command is to search the whitelist (so value=indicator), but it could also be used to search any list and then the value could be anything